airflow/www/views.py

```diff
@@ -1755,6 +1766,9 @@ def gantt(self, session=None):
             start_date = gantt_bar_item[1]
             end_date = gantt_bar_item[2]
             state = gantt_bar_item[3]
+            count = ti.try_number
```

Comment:
Do we have a reason to calculate count twice?

Reply:
Sorry. Looks like while resolving conflict,  it happened. Now deleted these chunks of lines.